test/button/button-icon.visual-diff.html

```diff
@@ -14,6 +14,7 @@
 	<meta charset="UTF-8">
 	<style>
 		.translucent-container {
+			background-color: orange;
```

Comment:
Ooooh, orange!

Reply:
Haha yeah - I think the background image loading previous had potential to throw things off.  Though, we are waiting till 0 connections.